tensorflow/python/keras/callbacks.py

```diff
@@ -1460,6 +1460,11 @@ class TensorBoard(Callback):
   You can find more information about TensorBoard
   [here](https://www.tensorflow.org/get_started/summaries_and_tensorboard).
 
+  Example:
```

Comment:
Hi, I've seen in most cases that "Usage Example " is used or nothing is used at all. However I may be wrong.

Reply:
I checked multiple classes in tf.estimator, tf.keras.Sequential, tf.keras.callbacks, etc. and they mostly use 'Example'. However, if you have a strong preference for 'Usage Example', let me know and I will channge it.

Reply:
haha... No I dont have any strong preference or bias toward "usage example" being used.😊
But if you wanted to follow the examples in the file they also had ```python``` in them, which, correct me if I misunderstand here, will stop them from running from doctests. What I'm trying to say here is that life can become infinitely easier for you if you follow the guidlines. I mean since they have all the rules to adding documentation to a function. do check it out at https://www.tensorflow.org/community/contribute/docs_ref . Neverthless my input to this pull request may be wildly irrelevant and unnecessary. Do let me know since I too am participating in the gci and am trying to figure all this out.🖖

Reply:
Hey, I know this is out of topic, but, where are all of the people talking. you know the IRC chat. Please let me know I've been dying to find out

Reply:
I have no clue, I don't even know if one exists. This github isn't the right forum to discuss about the IRC chat though :)

Reply:
PSA to the TensorFlow GCI folks, I've inquired multiple times to the mentors about making an IRC chat, but their stance is that they don't want to due to worries about code plagarism :(